test/resources/theme-ugly.css

```diff
@@ -1,5 +1,6 @@
 :root {
     --fd-color-action-1: magenta;
+    --fd-color-action: orangered;
```

Comment:
Looks like there are multiple other CSS variables you could put into the ugly theme.  Check the `scss/components/status-label.scss` file for other uses.